metrics/ci_integrations/lib/util/model/page.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:equatable/equatable.dart';
+
```

Comment:
Let's move this file to the Github Actions client package for now as it seems to be more package-specific. I suppose we can generalize it more later.